src/map/skill.cpp

```diff
@@ -19616,6 +20765,7 @@ int skill_detonator(struct block_list *bl, va_list ap)
 
 	block_list *src = va_arg(ap, block_list *);
 	skill_unit *unit = (skill_unit *)bl;
+	int flag = va_arg(ap, int);
```

Comment:
Since you changed the argument list size here for `skill_detonator()`, could you please also send 0 in the other call for RA_DETONATOR? There could be unforeseen random errors produced by not providing this missing argument, even though the compiler accepted it.